### ISSUE TITLE
Outdated Ubuntu Package

### ISSUE THREAD

**Issue by Godlikehobbit** [adnanh/webhook#243]
The prepackaged Ubuntu webhook is version 2.5.0 which appears to have the IP whitelist bug from #135

**Comment by hassanbabaie** [comment]
Hi @Godlikehobbit I think while you're waiting for someone to confirm you can use the following to get the latest?

`go get github.com/adnanh/webhook`

At least I think that his how I was doing it when not using the compiled file

Hass

**Comment by Godlikehobbit** [comment]
Yeah I already went ahead and fetched the latest version myself, just figured I'd open an issue specifically as the prepackaged version had the bug. Once I switched to 2.6.8 the same hooks.json file worked perfectly
